python310Packages.aioesphomeapi: 13.3.1 -> 13.4.0

fabaff opened the pull request:
Diff: https://github.com/esphome/aioesphomeapi/compare/refs/tags/v13.3.1...v13.4.0

Changelog: https://github.com/esphome/aioesphomeapi/releases/tag/v13.4.0

###### Description of changes

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

fabaff commented:
Result of `nixpkgs-review pr 217467` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>31 packages built:</summary>
  <ul>
    <li>esphome</li>
    <li>home-assistant-component-tests.airthings_ble</li>
    <li>home-assistant-component-tests.aranet</li>
    <li>home-assistant-component-tests.bluemaestro</li>
    <li>home-assistant-component-tests.bluetooth_adapters</li>
    <li>home-assistant-component-tests.bluetooth_le_tracker</li>
    <li>home-assistant-component-tests.bthome</li>
    <li>home-assistant-component-tests.esphome</li>
    <li>home-assistant-component-tests.fjaraskupan</li>
    <li>home-assistant-component-tests.govee_ble</li>
    <li>home-assistant-component-tests.homekit_controller</li>
    <li>home-assistant-component-tests.ibeacon</li>
    <li>home-assistant-component-tests.inkbird</li>
    <li>home-assistant-component-tests.kegtron</li>
    <li>home-assistant-component-tests.keymitt_ble</li>
    <li>home-assistant-component-tests.led_ble</li>
    <li>home-assistant-component-tests.moat</li>
    <li>home-assistant-component-tests.oralb</li>
    <li>home-assistant-component-tests.qingping</li>
    <li>home-assistant-component-tests.ruuvitag_ble</li>
    <li>home-assistant-component-tests.sensorpro</li>
    <li>home-assistant-component-tests.sensorpush</li>
    <li>home-assistant-component-tests.snooz</li>
    <li>home-assistant-component-tests.switchbot</li>
    <li>home-assistant-component-tests.thermobeacon</li>
    <li>home-assistant-component-tests.thermopro</li>
    <li>home-assistant-component-tests.tilt_ble</li>
    <li>home-assistant-component-tests.xiaomi_ble</li>
    <li>home-assistant-component-tests.yalexs_ble</li>
    <li>python310Packages.aioesphomeapi</li>
    <li>python311Packages.aioesphomeapi</li>
  </ul>
</details>

dotlambda reviewed (changes requested):
`postPatch` is no longer needed

fabaff committed:
python310Packages.aioesphomeapi: 13.3.1 -> 13.4.0

Diff: esphome/aioesphomeapi@refs/tags/v13.3.1...v13.4.0

Changelog: https://github.com/esphome/aioesphomeapi/releases/tag/v13.4.0

fabaff commented:
> postPatch is no longer needed

Thanks, removed.

fabaff commented:
Result of `nixpkgs-review pr 217467` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>31 packages built:</summary>
  <ul>
    <li>esphome</li>
    <li>home-assistant-component-tests.airthings_ble</li>
    <li>home-assistant-component-tests.aranet</li>
    <li>home-assistant-component-tests.bluemaestro</li>
    <li>home-assistant-component-tests.bluetooth_adapters</li>
    <li>home-assistant-component-tests.bluetooth_le_tracker</li>
    <li>home-assistant-component-tests.bthome</li>
    <li>home-assistant-component-tests.esphome</li>
    <li>home-assistant-component-tests.fjaraskupan</li>
    <li>home-assistant-component-tests.govee_ble</li>
    <li>home-assistant-component-tests.homekit_controller</li>
    <li>home-assistant-component-tests.ibeacon</li>
    <li>home-assistant-component-tests.inkbird</li>
    <li>home-assistant-component-tests.kegtron</li>
    <li>home-assistant-component-tests.keymitt_ble</li>
    <li>home-assistant-component-tests.led_ble</li>
    <li>home-assistant-component-tests.moat</li>
    <li>home-assistant-component-tests.oralb</li>
    <li>home-assistant-component-tests.qingping</li>
    <li>home-assistant-component-tests.ruuvitag_ble</li>
    <li>home-assistant-component-tests.sensorpro</li>
    <li>home-assistant-component-tests.sensorpush</li>
    <li>home-assistant-component-tests.snooz</li>
    <li>home-assistant-component-tests.switchbot</li>
    <li>home-assistant-component-tests.thermobeacon</li>
    <li>home-assistant-component-tests.thermopro</li>
    <li>home-assistant-component-tests.tilt_ble</li>
    <li>home-assistant-component-tests.xiaomi_ble</li>
    <li>home-assistant-component-tests.yalexs_ble</li>
    <li>python310Packages.aioesphomeapi</li>
    <li>python311Packages.aioesphomeapi</li>
  </ul>
</details>

dotlambda commented:
Result of `nixpkgs-review pr 217467` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>31 packages built:</summary>
  <ul>
    <li>esphome</li>
    <li>home-assistant-component-tests.airthings_ble</li>
    <li>home-assistant-component-tests.aranet</li>
    <li>home-assistant-component-tests.bluemaestro</li>
    <li>home-assistant-component-tests.bluetooth_adapters</li>
    <li>home-assistant-component-tests.bluetooth_le_tracker</li>
    <li>home-assistant-component-tests.bthome</li>
    <li>home-assistant-component-tests.esphome</li>
    <li>home-assistant-component-tests.fjaraskupan</li>
    <li>home-assistant-component-tests.govee_ble</li>
    <li>home-assistant-component-tests.homekit_controller</li>
    <li>home-assistant-component-tests.ibeacon</li>
    <li>home-assistant-component-tests.inkbird</li>
    <li>home-assistant-component-tests.kegtron</li>
    <li>home-assistant-component-tests.keymitt_ble</li>
    <li>home-assistant-component-tests.led_ble</li>
    <li>home-assistant-component-tests.moat</li>
    <li>home-assistant-component-tests.oralb</li>
    <li>home-assistant-component-tests.qingping</li>
    <li>home-assistant-component-tests.ruuvitag_ble</li>
    <li>home-assistant-component-tests.sensorpro</li>
    <li>home-assistant-component-tests.sensorpush</li>
    <li>home-assistant-component-tests.snooz</li>
    <li>home-assistant-component-tests.switchbot</li>
    <li>home-assistant-component-tests.thermobeacon</li>
    <li>home-assistant-component-tests.thermopro</li>
    <li>home-assistant-component-tests.tilt_ble</li>
    <li>home-assistant-component-tests.xiaomi_ble</li>
    <li>home-assistant-component-tests.yalexs_ble</li>
    <li>python310Packages.aioesphomeapi</li>
    <li>python311Packages.aioesphomeapi</li>
  </ul>
</details>
